src/storage/query/QueryBaseProcessor.inl

```diff
@@ -519,8 +519,7 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectEdgeProps(
         lastRank = rank;
         lastDstId = dstId;
         std::unique_ptr<RowReader> reader;
-        if (!onlyStructure
```

Comment:
As you said, I have a confusion:
`if ((!onlyStructure || retTTL.has_value()) && !val.empty())`   is satisfied,   but  `! onlyStructure` is `false`.
data ttl  has not expired  and `if (exp_ != nullptr) `  is satisfied. 
This will enter the `if (exp_! = Nullptr)` statement.   I don't know if the filter has any effect?

Reply:
If !onlyStructure is false, it means there are some props in value needed to return.
So we need to read the value.

Reply:
Got,  thx